packages/salesforcedx-vscode-core/src/commands/auth/orgLoginWebDevHub.ts

```diff
@@ -32,32 +32,32 @@ import {
 
 export class OrgLoginWebDevHubContainerExecutor extends OrgLoginWebContainerExecutor {
   public build(data: AuthDevHubParams): Command {
-    const command = new SfdxCommandBuilder().withDescription(
+    const command = new SfCommandBuilder().withDescription(
       nls.localize('org_login_web_authorize_dev_hub_text')
     );
 
     command
       .withArg(CLI.ORG_LOGIN_DEVICE)
       .withFlag('--alias', data.alias)
       .withArg('--set-default-dev-hub')
-      .withLogName('force_auth_device_dev_hub')
+      .withLogName('org_login_web_dev_hub_container')
```

Comment:
Same question as above - when did `device` start getting mapped to `container`?

Reply:
It's not that device maps to container, it's just that those 2 words are related and I prefer container as id for CB